crates/chainspec/src/api.rs

```diff
@@ -38,7 +39,7 @@ pub trait EthChainSpec: Send + Sync + Unpin + Debug {
     fn prune_delete_limit(&self) -> usize;
 
     /// Returns a string representation of the hardforks.
-    fn display_hardforks(&self) -> impl Display;
+    fn display_hardforks(&self) -> DisplayHardforks;
```

Comment:
a cool, yeah I had this in my backlog for a while

Reply:
done